### PR TITLE
fix broken docs link

### PR DESCRIPTION
**Pull request by navalev**

### CODE REVIEW

File: README.MD [hunk 1]
```diff
@@ -38,10 +38,10 @@ Presidio _(Origin from Latin praesidium ‘protection, garrison’)_ helps to en
 
 ## Installing Presidio
 
-1. [Using pip](https://microsoft.github.io/presidio/installation/#using-pip)
-2. [Using Docker](https://microsoft.github.io/presidio/installation/#using-docker)
-3. [From source](https://microsoft.github.io/presidio/installation/#install-from-source)
-4. [Supported Python Versions](https://microsoft.github.io/presidio/installation/#supported-python-versions)
+1. [Supported Python Versions](https://microsoft.github.io/presidio/installation/#supported-python-versions)
```
Comment (shiranr) [inline README.MD:41]: where is it suppose to refer to? the link just seems to go to installation.

Comment (navalev) [reply README.MD:41]: once we deploy the latest docs to github pages it will refer to the supported python versions section

**Review by shiranr — DISMISSED** [review]

**Review by balteravishay — APPROVED** [review]

**Comment by omri374** [comment]
Our readme is getting a little longer. How about using a badge for this? Like this one: 
[Python 3.6](https://www.python.org/downloads/release/python-360/) [Python 3.7](https://www.python.org/downloads/release/python-370/) [Python 3.8](https://www.python.org/downloads/release/python-380/)

**Comment by navalev** [comment]
> Our readme is getting a little longer. How about using a badge for this? Like these ones:
> [Python 3.6](https://www.python.org/downloads/release/python-360/) [Python 3.7](https://www.python.org/downloads/release/python-370/) [Python 3.8](https://www.python.org/downloads/release/python-380/)
> 
> Edit: If we specify which Python versions our analyzer supports, we can use this one (currently says 3 but it takes the version from the package on PyPI):
> [PyPI pyversions](https://pypi.python.org/pypi/presidio-analyzer/)
> 
> https://packaging.python.org/guides/distributing-packages-using-setuptools/#classifiers

Like this much better than adding a section i nthe docs, thanks @omri374 ! I would go with the first option, since we did got some attempts with python 3,9 and I wanted to highlight the we support 2.6 3.7 3.8

**Comment by SharonHart** [comment]
@omri374 Will it be rendered in mkdocs?

**Review by balteravishay — APPROVED** [review]

**Comment by navalev** [comment]
> @omri374 Will it be rendered in mkdocs? Or are we talking mentioning only in the main readme not in the docs?

@SharonHart I only added it to the main readme, not the github pages index file

**Review by omri374 — APPROVED** [review]

**Pull request merged**